### PR TITLE
doc: update network configuration for compute_subnet_cidr case

### PR DESCRIPTION
**Pull request by matteofiordarancio**
I verified with a test that for compute_subnet_cidr no Nat is needed

### CODE REVIEW

File: docs/networking.rst [hunk 1]
```diff
@@ -57,7 +57,7 @@ The configuration to use an existing private network requires the following sett
   master_subnet_id = subnet-<public>
   compute_subnet_id = subnet-<private>
 
-Both these configuration require to have a `NAT Gateway
+Note that this second configuration require to have a `NAT Gateway
```
Comment (enrico-usai) [inline docs/networking.rst]: Could you please verify if it works by using `use_public_ips = false`?

Comment (matteofiordarancio) [reply docs/networking.rst]: it does not work.

Comment (enrico-usai) [reply docs/networking.rst]: require --> requires

Comment (enrico-usai) [reply docs/networking.rst]: Thank you.
We could use the `..note::` syntax to show the message as an annotation:
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

Then: require --> requires

Comment (matteofiordarancio) [reply docs/networking.rst]: Done

File: docs/networking.rst [hunk 2]
```diff
@@ -57,9 +58,10 @@ The configuration to use an existing private network requires the following sett
   master_subnet_id = subnet-<public>
   compute_subnet_id = subnet-<private>
 
-Both these configuration require to have a `NAT Gateway
-<https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway.html>`_
-or an internal PROXY to enable web access for compute instances.
+.. note::
+    This second configuration require to have a `NAT Gateway
```
Comment (enrico-usai) [inline docs/networking.rst]: "This second configuration require to have a" --> "This second configuration requires a"

Comment (matteofiordarancio) [reply docs/networking.rst]: Done

**Pull request merged**